src/connection/connection_server_rules.cpp

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include "connection_server_rules.hpp"
+
+#include "server_type.hpp"
+#include "utility.h"
+#include "system_parameter.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
```

Comment:
Order each include group alphabetically:
```suggestion
#include "server_type.hpp"
#include "system_parameter.h"
#include "utility.h"

#include <assert.h>
#include <stdio.h>
#include <stdlib.h>
```